### DISCUSSION TITLE
关于账号提示被封禁

### DISCUSSION THREAD

**Discussion by QAbot-zh** [BurnHair/docs#155] [技术支持]
今天发现调用不是很顺利，然后检查模型调用的时候不是 100% 成功，失败的时候会提示用户被封禁（失败的时候连3.5都会报提示），账号也能登录，就有点迷惑，求解答。

![image](https://github.com/BurnHair/docs/assets/40236765/a6e9a1a9-d165-4306-9f97-418b66cf7dee)

我可以肯定我没有重复注册，也没有刻意搞 nsfw。注册以来实际用量不大，有放到我的 oneapi 里使用的行为，但仅出于使用方便，无商业分发，如果是商业分发那不可能只消耗这么一点，能查询我的使用记录进行验证，也没有使用沉浸式翻译等高并发行为。

![Clip_2024-07-09_19-35-32](https://github.com/BurnHair/docs/assets/40236765/94908e9b-f6a1-4ba5-8667-ce2e9d0ce186)

**Comment by QAbot-zh** [comment]
啊原来的额度全清空了？

![image](https://github.com/BurnHair/docs/assets/40236765/45a53851-d2b0-40fd-a56b-973144c51eb3)

**Reply by BennyThink** [reply]
你的用户状态异常，所以没有迁移额度

**Reply by QAbot-zh** [reply]
状态异常具体是指哪方面？我肯定没有重复注册，也没有刻意搞 nsfw，还有什么行为会导致异常

**Comment by QAbot-zh** [comment]
已通过邮件沟通后恢复，感谢站主，关帖
